src/vehicle.cpp

```diff
@@ -403,8 +403,11 @@ void vehicle::init_state( int init_veh_fuel, int init_veh_status )
     //Provide some variety to non-mint vehicles
     if( veh_status != 0 ) {
         //Leave engine running in some vehicles, if the engine has not been destroyed
+        //chance decays from 1 in 4 vehicles on day 0 to 1 in (day + 4) in the future.
+        int current_day = std::max( to_days<int>( calendar::turn - calendar::turn_zero ), 0 );
```

Comment:
I think we should better use `calendar::start_of_cataclysm` instead of `calendar::turn_zero` here. Scenarios can override starting date, so if we start, for example, "The Next Summer" scenario, on turn 0 of this scenario we still have a decent chance to find a running vehicle despite ~9 months past the Cataclysm. The same for non-default (later) starting dates for ordinary scenarios.